docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1401 @@
+{
```

Comment:
<p>I think that in addition to this toy example, we should start considering testing this on other (labeled) datasets:</p><p> </p><p>https://wu.renjie.im/research/anomaly-benchmarks-are-flawed/#ucr-time-series-anomaly-archive</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
Thanks for  sharing the link. I think I have some data on my pc already as well. I am going to dig into the datasets and will let you know.

So, after revising the code after this round of comments, I will use `_murlin` / `murlin` to find anomalies on other data sets (and compare them with STUMPY MP)

Reply:
Thank you!

Reply:
I compared the results of `_murlin` and STUMPY on taxi data. We can add more comparisons later.

> on other (labeled) datasets

**Is it important to have `labeled` data?**  Because, for one, it was hard for me to find data sets with labels, and I read it is hard to trust ground truth labels in anomaly detection as they usually miss some anomalies!), and for another, the goal is to show the results of  `_murlin` and `STUMPY matrix profile approach` are the same.  

So, are you trying to figure out if (STUMPY) matrix profile is a good approach for finding anomaly? Because, that is a different story.

Reply:
I am more interested in using a dataset that is more visually interesting and less trivial. That is my goal

Reply:
I see. All good then! 😊